network-api/networkapi/wagtailpages/customblocks.py

```diff
@@ -180,6 +180,11 @@ class FigureBlock2(blocks.StructBlock):
         required=False,
         help_text='Optional URL that this figure should link out to.',
     )
+    square_image = blocks.BooleanBlock(
+        default=True,
+        required=False,
+        help_text='Note: If checked, images will be square.'
```

Comment:
Can you add more detail to the help text? Something that indicates the image will be cropped if square, and perhaps explicitly stating all images will have the same width otherwise?

Reply:
Sure can.